test/Elastica/Aggregation/AdjacencyMatrixTest.php

```diff
@@ -68,21 +68,6 @@ public function testToArray()
         $this->assertEquals($expected, $agg->toArray());
     }
 
-    /**
-     * @group unit
-     */
-    public function testNonStringNamedFilters()
```

Comment:
Here's the test in question ;)

Reply:
yea, just remove the test. PHP is taking care of passing a string to the method and throwing exceptions otherwise